### PR TITLE
Define and rely on "display-capture" permission-based feature policy.

### PR DESCRIPTION
**Pull request by jan-ivar**
Fix for https://github.com/w3c/mediacapture-screen-share/issues/39.

### CODE REVIEW

**Comment by martinthomson** [comment]
The string "display" in isolation means something different to what you intend it to mean.  I think that you want "display-capture" or something along those lines.

**Comment by jan-ivar** [comment]
@martinthomson FWIW `"display"` is the name we've been discussing since https://github.com/w3c/mediacapture-screen-share/issues/28#issuecomment-417339077, and is at this point already in [permissions](https://w3c.github.io/permissions/#screen-capture) and [getDisplayMedia](https://w3c.github.io/mediacapture-screen-share/#dom-mediadevices-getdisplaymedia). How strongly do you feel about it?

File: index.html [hunk 1]
```diff
@@ -296,8 +270,7 @@ <h2>
                       Failure</em> below.</p>
                   </li>
                   <li>
-                    <p>For the origin identified by
```
Comment (martinthomson) [inline index.html:299]: I assume that this is subsumed by the "prompt the user to choose" algorithm.

**Comment by martinthomson** [comment]
I'd prefer to get the right name.  I don't want to take a name that might be used for say access to displays.

**Comment by jan-ivar** [comment]
@martinthomson @alvestrand discuss. What would "access to displays" mean? We don't say "camera-capture" FWIW.

File: index.html [hunk 2]
```diff
@@ -884,6 +857,34 @@ <h2 id="deviceId">
...
+          id="ref-for-concept-document①⓪">document</a>'s
+        <a data-link-type="dfn"
+          href="https://html.spec.whatwg.org/multipage/dom.html#concept-document-feature-policy"
+          id="ref-for-concept-document-feature-policy">feature policy</a>
+        determines whether any content in that document is allowed to use
+        <code>getDisplayMedia</code>. If disabled in any document, no content in
+        the document will be
+        <a data-link-type="dfn"
+          href="https://html.spec.whatwg.org/multipage/iframe-embed-object.html#allowed-to-use"
+          id="ref-for-allowed-to-use②">allowed to use</a>
```
Comment (alvestrand) [inline index.html]: Is the circled 2 on this line an accidental character?

**Comment by alvestrand** [comment]
The main purpose of a display is to display stuff (facing out of the computer).
We could imagine "display" being used for "is allowed to display on the displayed surface". But it's hard to imagine a Web page without that right, so it's kind of like allowing a fish to be wet.

The permission we're mediating here is a secondary function - that of allowing the display to be scraped by the computer.

The closest permission analogy to "display" in the other direction I can think of is "speaker"; there, we mediate an outward-facing access; we never capture from a speaker.

The only reason to stick with "display" is that we're already using it, and nobody else is. I'm not greatly worried about it either way.

File: index.html (hunk 2 above)
Comment (jan-ivar) [reply index.html]: Boiler-plate from the fullscreen spec I copied. Removed now. Thanks for spotting it!

**Comment by jan-ivar** [comment]
So which do we want: `display`, `display-capture`, `screen-capture`, `display-media`?

It's a bit odd, the spec itself is titled "Screen capture", but the term is mostly used [outside this spec](https://w3c.github.io/permissions/#screen-capture). "display capture" appears unused, but would seem logical even though search engines think it a [typo](https://duckduckgo.com/?q=display+capture).

**Comment by jan-ivar** [comment]
I picked one.

File: index.html [hunk 3]
```diff
@@ -884,6 +857,34 @@ <h2 id="deviceId">
...
       </section>
     </section>
+    <section>
+      <h1 id=feature-policy-integration>Feature Policy Integration</h1>
+      <p>This specification defines a <a data-link-type="dfn"
+        href="https://wicg.github.io/feature-policy/#policy-controlled-feature"
+        >policy-controlled feature</a>
+      identified by the string
+      <code>"<dfn data-lt="display-capture-feature">display-capture</dfn>"</code>.
+      It's <a data-link-type="dfn"
```
Comment (martinthomson) [inline index.html]:
```suggestion
      Its <a data-link-type="dfn"
```

**Review by martinthomson — APPROVED** [review]

**Comment by henbos** [comment]
@alvestrand to review this before merging.

**Review by alvestrand — APPROVED** [review]

**Pull request merged**